change-validation/src/main/scala/com/normation/plugins/changevalidation/ChangeRequestJson.scala

```diff
@@ -439,10 +436,11 @@ object DirectiveChangeActionJson {
           case d: DirectiveDeleteChangeJson => DirectiveChangeActionJson(ActionChangeJson.delete, d)
           case d: DirectiveModifyChangeJson => DirectiveChangeActionJson(ActionChangeJson.modify, d)
         }) match {
-        case Empty                          =>
-          Result.fromErrorString(s"Error while serializing directives from CR ${change.firstChange.diff.directive.id.serialize}")
-        case Failure(msg, exception, chain) => Result.fromErrorString(msg)
-        case Full(value)                    => value
+        case Left(err)    =>
+          Result.fromErrorString(
+            s"Error while serializing directives from CR ${change.firstChange.diff.directive.id.serialize}" + err.msg
```

Comment:
There is a little issue in the string here : 
```suggestion
            s"Error while serializing directives from CR ${change.firstChange.diff.directive.id.serialize}: ${err.msg}"
```